src/appliance_setup/avs/avsarconboarder/retriever/vsphere/vsphere_details_retriever.py

```diff
@@ -1,24 +1,104 @@
+import os
+import json
+from urllib.parse import urlparse
+import logging
 from ...entity._vsphere_resource import VSphereResourceData
 from ...retriever._retriever import Retriever
-
+from pkgs._govc_cli import govc_cli
+from pkgs._exceptions import vCenterOperationFailed
 
 class VSphereDetails(Retriever):
     instance = None
+    _data_center = "SDDC-Datacenter"
     _arc_rp_name = "arc-rp"
     _arc_folder_name = "arc-folder"
-    _data_center = "SDDC-Datacenter"
-    _data_store = "vsanDatastore"
     _template_name = "arc-template"
-    _cluster_name = "Cluster-1"
```

Comment:
NIT : Keep these definitions here and use these variables below rather than using hardened strings in the code directly.

Reply:
I will take this up in next PR as this does not change any functionality.